qa-brh.planx-pla.net/portal/gitops.json

```diff
@@ -165,9 +165,31 @@
         }
       },
       "authorization": {
-        "enabled": true
+        "enabled": true,
+	"columnTooltip": "Filter by data access",
+        "supportedValues": {
+          "accessible": {
+            "enabled": true,
+            "menuText": "Available"
+          },
+          "unaccessible": {
+            "enabled": false,
+            "menuText": "Not Accessible"
+          },
+          "pending": {
+            "enabled": false,
+            "menuText": "Pending"
+          },
+          "notAvailable": {
+            "enabled": true,
+            "menuText": "Not Available"
+          }
+        }
       }
     },
+    "discoveryUseAggWTS": {
+	    "enabled": true
+    },
```

Comment:
@johnfrancismccann not sure if this is the correct addition to the BRH config. Can you please review?